docs/tutorials/filtering_large_catalogs.ipynb

```diff
@@ -249,6 +250,28 @@
     "We can stack a several number of filters, which are applied in sequence. For example, `catalog.box_search().polygon_search()` should result in a perfectly valid HATS catalog containing the objects that match both filters."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "9a573c7c",
+   "metadata": {},
+   "source": [
+    "## Selecting data by column query\n",
+    "\n",
+    "We may also filter by columns via query().\n",
+    "\n",
+    "The expression inside () follows the same syntax accepted by Pandas .query(), which supports a subset of Python expressions for filtering DataFrames (e.g., column comparisons, logical operators, and basic functions)."
```

Comment:
In long term I would probably want to have more about Pandas.query() language. In the short term, can you find some links to materials that could be usueful to users?